rllib/algorithms/algorithm_config.py

```diff
@@ -361,6 +360,7 @@ def __init__(self, algo_class=None):
         self.input_evaluation = DEPRECATED_VALUE
         self.horizon = DEPRECATED_VALUE
         self.soft_horizon = DEPRECATED_VALUE
+        self.no_done_at_end = DEPRECATED_VALUE
```

Comment:
Also removed this config option now: Should be replaced by using `truncated` (instead of `done`) being returned by an env's `step()`.